Demo/src/main/java/com/paypal/android/ui/card/validation/CardType.kt

```diff
@@ -0,0 +1,30 @@
+package com.paypal.android.ui.card.validation
+
+@Suppress("MagicNumber")
+enum class CardType(
```

Comment:
We should be able to reuse this enum once we look at supporting Card Fields in the SDK, but for now, I kept this as simple as possible to only handle formatting card numbers and max card lengths.